core/src/value/partial.rs

```diff
@@ -173,42 +183,45 @@ enum DicomTimeImpl {
     Fraction(u8, u8, u8, u32, u8),
 }
 
-/// Represents a Dicom DateTime value with a partial precision,
+/// Represents a Dicom date-time (DT) value with a partial precision,
 /// where some date or time components may be missing.
 ///
-/// `DicomDateTime` is always internally represented by a [DicomDate]
-/// and optionally by a [DicomTime].
+/// `DicomDateTime` is always internally represented by a [DicomDate].
+/// The [DicomTime] and a timezone [FixedOffset] values are optional.
 ///
-/// It implements [AsRange] trait and also holds a [FixedOffset] value, from which corresponding
-/// [datetime][DateTime] values can be retrieved.
+/// It implements [AsRange] trait, which serves to retrieve a [PreciseDateTimeResult] from values
+/// with missing components.
 /// # Example
 /// ```
 /// # use std::error::Error;
 /// # use std::convert::TryFrom;
 /// use chrono::{DateTime, FixedOffset, TimeZone, NaiveDateTime, NaiveDate, NaiveTime};
-/// use dicom_core::value::{DicomDate, DicomTime, DicomDateTime, AsRange};
+/// use dicom_core::value::{DicomDate, DicomTime, DicomDateTime, AsRange, PreciseDateTimeResult};
 /// # fn main() -> Result<(), Box<dyn Error>> {
 ///
 /// let offset = FixedOffset::east_opt(3600).unwrap();
 ///
-/// // the least precise date-time value possible is a 'YYYY'
-/// let dt = DicomDateTime::from_date(
+/// // lets create the least precise date-time value possible 'YYYY' and make it time-zone aware
+/// let dt = DicomDateTime::from_date_with_time_zone(
 ///     DicomDate::from_y(2020)?,
 ///     offset
 /// );
+/// // the earliest possible value is output as a [PreciseDateTimeResult]
```

Comment:
This is inside a doctest.

```suggestion
/// // the earliest possible value is output as a PreciseDateTimeResult
```